content/tutorials/publish-subscribe.textile

```diff
@@ -293,22 +293,37 @@ blang[android].
   "See this step in Github":https://github.com/ably/tutorials/commit/pubsub-android-step2
 
 blang[javascript].
-  To start using Ably in your web app, you first need to include the Ably library. We recommend that you include the latest client library from our CDN using a simple @<script>@ tag. The client library must be instantiated with the API key you copied in Step 1. Note in production we recommend you always use the "token authentication scheme":/core-features/authentication#token-authentication for browser clients, however in this example we use an API key for simplicity.
+  First, create a blank HTML page called @example.html@, to host your web app:
+
+  ```[html]
+      <!DOCTYPE html>
+      <html lang="en">
+        <head>
+          <meta charset="UTF-8" />
+          <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+          <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+          <title>Ably Pub Sub</title>
+        </head>
+        <body>
+        </body>
+      </html>
+  ```
+
+  To start using Ably in your web app, you first need to include the Ably client library. In this example, we recommend that you include the latest client library from our CDN using a simple @<script>@ tag. The client library must be instantiated with the API key you copied in Step 1.
+  
+  bq. **Note**: in production we recommend you always use the "token authentication scheme":/core-features/authentication#token-authentication for browser clients, however in this example you will use an API key for simplicity.
 
-  Include the code below just before your closing your HTML @</html>@ tag.
+  Include the code below just before your closing @</html>@ tag.
 
   ```[javascript]
-    <!-- Include the latest Ably Library  -->
-    <script src="https://cdn.ably.io/lib/ably.min-1.js"></script>
+  <script src="https://cdn.ably.io/lib/ably.min-1.js"></script>
```

Comment:
I think this should be `https://cdn.ably.io` -> `https://cdn.ably.com`. I think use `.com`, but please double-check.